Description: use qWaitForWindowExposed instead of qWaitForWindowActive
 The qWaitForWindowActive() function returns false when the test is
 run inside Xvfb and there is no window manager.
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: no
Last-Update: 2019-04-21

--- a/tests/auto/quicktest/signalspy/tst_signalspy.cpp
+++ b/tests/auto/quicktest/signalspy/tst_signalspy.cpp
@@ -73,7 +73,7 @@ void tst_SignalSpy::testCount()
     window.resize(200, 200);
     window.setSource(testFileUrl("signalspy.qml"));
     window.show();
-    QVERIFY(QTest::qWaitForWindowActive(&window));
+    QVERIFY(QTest::qWaitForWindowExposed(&window));
     QVERIFY(window.rootObject() != nullptr);
 
     QObject *mouseSpy = window.rootObject()->findChild<QObject*>("mouseSpy");
